Source/init.cpp

```diff
@@ -173,7 +173,7 @@ void init_archives()
 			message.append(paths[i]);
 			message += '\'';
 		}
-		LogVerbose("MPQ search paths%s", message.c_str());
+		LogVerbose("MPQ search paths {}", message);

```

Comment:
This shouldn't have a space, `message` starts with a newline

Reply:
Why not `MPQ search paths:` then?